susemanager/susemanager.changes

```diff
@@ -1,3 +1,4 @@
+- mgr-sync: Raise a proper exception when duplicated lines exist in a config file (bsc#1182742)
```

Comment:
Minor nipick: when your entry is the first one after the last headers, add also a line break:
```suggestion
- mgr-sync: Raise a proper exception when duplicated lines exist in a config file (bsc#1182742)

```

Reply:
Thanks for the note.